frontend/packages/db-structure/src/parser/drizzle/convertToPgType.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * Convert Drizzle column types to PostgreSQL column types
+ * ref: https://orm.drizzle.team/docs/column-types/pg
+ */
+export const convertDrizzleTypeToPgType = (
+  drizzleType: string,
+  options?: Record<string, unknown>,
+): string => {
+  switch (drizzleType) {
```

Comment:
IMO: it looks like case and return are the same thing and should be grouped together in default.

Reply:
You were right. I refactored it!
https://github.com/liam-hq/liam/pull/2299/commits/94728d20308c2c47287142f8595359540b0e6d64